packages/data-schema/__tests__/CustomOperations.test-d.ts

```diff
@@ -858,3 +861,22 @@ describe('.for() modifier', () => {
     a.subscription().for(a.ref('Model'));
   });
 });
+
+describe('.arguments() modifier', () => {
+  // Test to verify that CustomType can be used as an argument in custom operations
+  it('accepts CustomType in arguments', () => {
+    const operation: CustomOperation<any, "arguments" | "for", "queryCustomOperation"> = a.query().arguments({
+      customArg: a.customType({
+        field1: a.string(),
+        field2: a.integer()
+      })
+    });
+  });
```

Comment:
Would it be worth adding some type comparison testing around `Schema['fcnCall']['functionHandler']`?

It appears to work as I would expect, but looking at this test, it covers the definition, but doesn't test the resulting types.

In my example app, I leveraged the types for the handler definition like:
```
export const handler: Schema['fcnCall']['functionHandler'] = async (x) => {
    return {
        todoCount: (await client.models.Todo.list()).data.length,
        x: JSON.stringify(x.arguments),
    }
}

// Picking into the type to inspect the arg types
type T = Parameters<Schema['fcnCall']['functionHandler']>[0]['arguments']['arg1']
```